packages/sample-app/sample_app/traceloop_openai_tools.py

```
@@ -0,0 +1,327 @@
+import os
+import json
+import requests
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, Any, List
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)